.github/workflows/test.yml

```diff
@@ -21,7 +21,10 @@ jobs:
         key: venv-${{ runner.os }}-${{ matrix.python-version }}-${{ hashFiles('**/requirements*.txt') }}
 
     - name: Install python dependencies
-      run: make install
+      run: |
+        python -m venv venv
```

Comment:
Is there a reason not to use `make virtualenv` or to make this change in the `Makefile`?

Reply:
Oh yeah, I was just being lazy because I'm going to replace the Makefile in a following PR 😅

Reply:
Changed the Makefile instead now